sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeBatchActionsResult.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.AI.TextAnalytics.Models;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Determine the results from analyze batch actions in collections of KeyphraseExtraction, RecognizeEntities and RecognizePiiEntities.
+    /// </summary>
+    public class AnalyzeBatchActionsResult
+    {
+        internal AnalyzeBatchActionsResult(IReadOnlyCollection<ExtractKeyPhrasesActionResult> extractKeyPhrasesActionResult, IReadOnlyCollection<RecognizeEntitiesActionResult> recognizeEntitiesActionResults, IReadOnlyCollection<RecognizePiiEntitiesActionResult> recognizePiiEntitiesActionResults, TextDocumentBatchStatistics statistics)
+        {
+            ExtractKeyPhrasesActionsResults = extractKeyPhrasesActionResult;
```

Comment:
should change `extractKeyPhrasesActionResult` to `extractKeyPhrasesActionResults` with plural.

Reply:
Do we need to keep the word `Action` in there? I am not using it in JS because I think it is unnecessary.

Reply:
For the sake of this PR, let's leave it as is. After the release we can go back to naming conversations